script/1_0_2current_en_script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -1,23 +1,23 @@
 <Page1>
 I wake up absentmindedly.
-The arm reaching out to the glasses feels slightly heavy.
-I put on my glasses, wondering if it's a slight fever like it was someone else's business.
+My arm reaching out to my glasses feels slightly heavy.
+I put on my glasses, wondering if I had a slight fever like it was someone else's business.
 <Page2>
 Even as my vision returns to normal, I still feel a slight dizziness.
-It resembles the symptoms of a cold, but this is more like a morning fever.
-It's like stuck engine heated on a flame which gets alleviated simply through the passage of time.
+It may resemble the symptoms of a cold, but this is more like a morning fever.
+It's just like a halted engine being warmed up by a fire, it'll subside with time.
```

Comment:
I'm not sure halting is right and google seems to think I mean "stalling" when I google it. I think the analogy here is that he's like a car that hasn't started up in a while.
> 動かないエンジンを火にくべて暖めるようなもので、時間さえ経てば治まってくれる。
Like an engine that hasn't started up in a while being warmed by combustion, this feeling will subside with time.